### DISCUSSION TITLE
1.8.9 Textures

### DISCUSSION THREAD

**Discussion by 9182736450typist** [craftycodie/SpriteCaster#37] [Ideas]
Can you add support for resource packs in newer versions such as 1.8.9, 1.5.2, 1.4.7 and 1.2.5?

**Comment by craftycodie** [comment]
It's technically possible, I don't have time to do it but if you'd like to feel free to PR itOn 1 Sept 2022 05:00, 9182736450typist ***@***.***> wrote:
Can you add support for resource packs in newer versions such as 1.8.9, 1.5.2, 1.4.7 and 1.2.5?
